tests/test_tasks.py

```diff
@@ -419,69 +417,58 @@ def setUp(self):
-        self.assertEqual(handler._notify(10), True)
-        self.clear_db_config()
-
-    def test_is_ok_to_notify(self):
-        handler = self.get_handler()
-        count_unreviwed_notes = 10
-        self.assertEqual(handler._is_ok_to_notify(count_unreviwed_notes), False)
-        admin_email = 'inna-testing@gmail.com'
-        thres_unreviewed_notes = 5
-        config.set_for_repo('haiti', admin_user_email=admin_email)
-        config.set_for_repo('haiti', thres_unreviewed_notes=thres_unreviewed_notes)
-        self.assertEqual(handler._is_ok_to_notify(count_unreviwed_notes), True)
-        thres_unreviewed_notes = 10
-        config.set_for_repo('haiti', admin_user_email=admin_email)
-        config.set_for_repo('haiti', thres_unreviewed_notes=thres_unreviewed_notes)
-        self.assertEqual(handler._is_ok_to_notify(count_unreviwed_notes), True)
-        thres_unreviewed_notes = 20
-        config.set_for_repo('haiti', admin_user_email=admin_email)
-        config.set_for_repo('haiti', thres_unreviewed_notes=thres_unreviewed_notes)
-        self.assertEqual(handler._is_ok_to_notify(count_unreviwed_notes), False)
-        self.clear_db_config()
-
-    def test_get_admin_user_email(self):
-        handler = self.get_handler()
-        self.assertEqual(handler._get_admin_user_email(), None)
-        admin_email = 'inna-testing@gmail.com'
-        config.set_for_repo('haiti', admin_user_email=admin_email)
-        self.assertEquals(handler._get_admin_user_email(), admin_email)
-        self.clear_db_config()
-        self.assertEqual(handler._get_admin_user_email(), None)
-
-    def test_get_thres_unreviewed_notes(self):
-        handler = self.get_handler()
-        self.assertEquals(handler._get_thres_unreviewed_notes(),
-                          const.DEFAULT_THRES_NUM_UNREVIEWED_NOTES)
-        thres_unreviewed_notes = 50
-        config.set_for_repo('haiti', thres_unreviewed_notes=thres_unreviewed_notes)
-        self.assertEquals(handler._get_thres_unreviewed_notes(),
-                          thres_unreviewed_notes)
-        self.clear_db_config()
-        self.assertEquals(handler._get_thres_unreviewed_notes(),
-                          const.DEFAULT_THRES_NUM_UNREVIEWED_NOTES)
\ No newline at end of file
+    def test_maybe_notify(self):
+        # _maybe_notify fails if unreviewed_notes_email is set to ''
+        config.set(unreviewed_notes_email='',
+                   unreviewed_notes_email_threshold='100')
+        self.assert_(not self.handler._maybe_notify(10))
+
+        config.set(unreviewed_notes_email='inna-testing@gmail.com',
+                   unreviewed_notes_email_threshold='10')
+        self.assert_(self.handler._maybe_notify(10))
+
+    def test_should_modify(self):
+        # Default settings
+        config.set(unreviewed_notes_email='',
+                   unreviewed_notes_email_threshold='100')
+        self.assert_(not self.handler._should_modify(10))
```

Comment:
You should also test a case where the number of emails is more than the threshold, but unreviewed_notes_email is empty. If you add that, you may not need this test at line 441-443 instead.